docs/developer/getting-started/create-edit-forms.md

```diff
@@ -0,0 +1,101 @@
+
+# Alternative Create/Edit Forms 
+This section describes how to make create/edit forms without CruResource or the CreateEditView mixin.
+
+There may be a situation where you need to make a create/edit form that doesn’t automatically get the resource value passed into it because the component is located outside of the same `/edit` folder as the other create/edit forms. In this case, you will need to manually create the skeleton resource from a class model and use that as the form value so that the methods passed into AsyncButton have access to enough information to save the resource.
+
+There may be a situation where you need Rancher to create and edit data that is not a Kubernetes resource. But even in this situation, we recommend using the same basic architecture as is used for saving Kubernetes resources. The reason is that many users want to automate their workflows with GitOps, and they want to be able to manipulate data with both config files and with forms in the Rancher UI. This means that even if it is not currently possible to manage the resource in YAML, that ability may be added later because of the industry’s overarching need for automation and GitOps. The new Rancher UI will need to be able to consistently support GitOps workflows.
+
+A custom create component would typically need at least these imports:
+
+```
+import AsyncButton from '@/components/AsyncButton';
+import { _CREATE } from '@/config/query-params';
+import { EPINIO_TYPES } from '@/products/epinio/types';
+import { exceptionToErrorsArray } from '@/utils/error';
+```
+
+The AsyncButton should be used to save the resource, and it will show a loading message if the resource takes time to be saved. The below example shows a minimal usage of AsyncButton:
+
+```
+<AsyncButton
+  :disabled="!validationPassed"
+  :mode="mode"
+  @click="onSubmit"
+/>
+```
+
+You can maintain the form validation state in `data()`:
+
+```
+data() {
+    return {
+      errors:           [],
+      value:           { name: '' },
+      submitted:       false,
+      mode:            _CREATE
+    };
+  },
+```
+
+You will also need to provide the resource schema so that it can be used by `save()` to create the resource by calling the appropriate back end route for creating the resource.
+
+When a skeleton resource is not passed into your create/edit form as a value, you can create the new resource instance with a `dispatch` to the store as soon as the form is rendered. Then you can use it as the initial value for your form. An example of calling the store to create the initial resource value is below:
+
+```
+this.value = await this.$store.dispatch(`epinio/create`, { type: EPINIO_TYPES.NAMESPACE });
+```
+
+When the resource is saved, it should call a `save` method. This can be the `save`defined in the CreateEditView mixin, or it can be a `save` method defined on the resource model, which overrides the mixin:
+
+```
+onSubmit() {
```

Comment:
This example needs updating given changes to the other PR (should be async given save and also handle an asyncButton's cb)

Reply:
Changed it to use this example from PromptChangePassword and clarified that that method needs to go in the form, not the model.

```
async submit(buttonCb) {
  try {
    await this.$refs.changePassword.save();
    this.show(false);
    buttonCb(true);
  } catch (err) {
    buttonCb(false);
  }
}
```